siem_integrations/clx_query_service/conf/clx_blz_reader_conf.yaml

```diff
@@ -0,0 +1,12 @@
+input_format: csv
+tables:
+ - 
+   input_path: "/rapids/my_data/ml-25m/movies.csv"
```

Comment:
Are there directions on how to configure this (e.g., the location to the data)?

Reply:
It is just providing absolute path of the dataset and suitable table name. Added more information in the `readme.md`